tests/unit/jobserver/views/test_status.py

```diff
@@ -172,3 +172,13 @@ def test_status_counts_all_pending_jobs(rf):
 
     output = first(response.context_data["backends"])
     assert output["queue"]["pending"] == 3
+
+
+def test_status_shows_only_active_backends(rf):
+    BackendFactory.create_batch(2, is_active=True)
+    BackendFactory(is_active=False)
+
+    request = rf.get("/")
+    response = Status.as_view()(request)
+
+    assert len(response.context_data["backends"]) == 2
```

Comment:
I've just added `set_from_list` to `jobserver/utils.py` as part of pulling querying out of rendering.  So you can improve this assertion to check the backends are the ones you expected.

Assign the output of `create_batch()` to a variable called `backends`, then this line can become:

```
assert set_from_qs(response.context_data["backends"]) == set_from_list(backends)
```